src/main/java/server/DeleteRequestHandler.java

```diff
@@ -0,0 +1,27 @@
+package server;
+
+import HTTPComponents.StatusCode;
+
+import java.io.File;
+
+public class DeleteRequestHandler implements RequestHandler {
+
+    public Response handle(Request request) {
+        if (delete(request.getResourceIdentifier())) {
+            return new ResponseBuilder()
+                    .setStatusCode(StatusCode.NO_CONTENT)
+                    .build();
+        } else {
```

Comment:
Do you know what guard clauses are? We typically favor those over `else`

Reply:
in this case since we don't even need the else since we return early if the condition matches

Reply:
Got it! That makes so much sense! Fixed.